extension/encoding/googlecloudlogentryencodingextension/internal/vpcflowlog/parser.go

```diff
@@ -0,0 +1,329 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package vpcflowlog // import "github.com/open-telemetry/opentelemetry-collector-contrib/extension/encoding/googlecloudlogentryencodingextension/internal/vpcflowlog"
+
+import (
+	"fmt"
+
+	gojson "github.com/goccy/go-json"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	semconv "go.opentelemetry.io/otel/semconv/v1.34.0"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/extension/encoding/googlecloudlogentryencodingextension/internal/shared"
+)
+
+const (
+	// VPC flow log name suffixes for detection
+	VPCFlowLogNameSuffix        = "networkmanagement.googleapis.com%2Fvpc_flows"
+	ComputeVPCFlowLogNameSuffix = "compute.googleapis.com%2Fvpc_flows"
+
+	// Connection fields
+	gcpVPCFlowReporter       = "gcp.vpc.flow.reporter"
+	gcpVPCFlowBytesSent      = "gcp.vpc.flow.bytes_sent"
+	gcpVPCFlowPacketsSent    = "gcp.vpc.flow.packets_sent"
+	gcpVPCFlowStartTime      = "gcp.vpc.flow.start_time"
+	gcpVPCFlowEndTime        = "gcp.vpc.flow.end_time"
+	gcpVPCFlowNetworkRTTMsec = "gcp.vpc.flow.network.rtt_msec"
```

Comment:
```suggestion
	gcpVPCFlowNetworkRTTMsec = "gcp.vpc.flow.network.rtt_ms"
```

To respect the usual time suffix for duration

Reply:
Updated this, the readme, and even the variable name here